docs/ocid.8.md

```diff
@@ -54,7 +54,7 @@ ocid is meant to provide an integration path between OCI conformant runtimes and
   path to configuration file
 
 **--conmon**=""
-  path to the conmon executable (default: "/usr/libexec/ocid/conmon")
+  path to the conmon executable (default: "/usr/local/libexec/ocid/conmon")
```

Comment:
I think we should make this change as we can always fixup the config in the rpm. This is something that needs to setup during dev explicitly as it is out of sync.